common/centraldashboard/base/kustomization.yaml

```diff
@@ -35,6 +35,20 @@ vars:
     apiVersion: v1
   fieldref:
     fieldpath: data.clusterDomain
+- name: userid-header
+  objref:
+    kind: ConfigMap
+    name: parameters
```

Comment:
i believe this will remain 'parameters' since no prefix is specified.
We should probably have an e2e that makes sure configmap names are unique per manifest.

Reply:
Does this have any adverse effects right now?
Or can this PR go in?

Reply:
IIUC that is independent of this PR. I don't believe this PR is adding the ConfigMap for central dashboard. And this manifest currently doesn't add a common prefix. So there's no guarantee the config map is going to be unique and it could conflict with configmaps from other applications. But I think that's already the case and unrelated to this PR.

Should we open an issue to track that and fix later?

Reply:
@jlewi that is correct, the CentralDashboard ConfigMap isn't introduced by this PR.
I opened https://github.com/kubeflow/manifests/issues/288 to track the issue.